lua/starfall/libs_sh/entities.lua

```diff
@@ -1569,4 +1570,78 @@ function ents_methods:getNWVarTable()
 	return instance.Sanitize(getent(self):GetNWVarTable())
 end
 
+if DarkRP then
```

Comment:
These need to be in the DarkRP library, not the entities library. Take a look at how the 'Wire' lib extends the entities library as example.